Index: js/script.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/script.js b/js/script.js
new file mode 100644
--- /dev/null	(date 1748162284592)
+++ b/js/script.js	(date 1748162284592)
@@ -0,0 +1,2 @@
+'use strict';
+
Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
new file mode 100644
--- /dev/null	(date 1748162284583)
+++ b/index.html	(date 1748162284583)
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
\ No newline at end of file
