Index: js/script.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/script.js b/js/script.js
new file mode 100644
--- /dev/null	(date 1748169296889)
+++ b/js/script.js	(date 1748169296889)
@@ -0,0 +1,13 @@
+    'use strict';
+
+    var sign = prompt("What's your sign?");
+
+    if (sign.toLowerCase() == "scorpio") {
+        alert("Wow! I'm a Scorpio too!");
+    }
+
+    // есть множество способов использовать prompt
+    var sign = window.prompt(); // открывается пустое окошко с запросом ввода текста
+    var sign = prompt(); //  открывается пустое окошко с запросом ввода текста
+    var sign = window.prompt("Are you feeling lucky"); // открывается окошко с текстом "Are you feeling lucky"
+    var sign = window.prompt("Are you feeling lucky", "sure"); // открывается окошко с текстом "Are you feeling lucky" и значением по умолчанию "sure"
